services/web/server/src/simcore_service_webserver/folders/_folders_db.py

```diff
@@ -188,28 +213,50 @@ async def get_for_user_or_workspace(
         return FolderDB.from_orm(row)
 
 
+_MAX_ITEMS_UPDATE: Final = 100
+
+
 async def update(
     app: web.Application,
     *,
-    folder_id: FolderID,
-    name: str,
-    parent_folder_id: FolderID | None,
+    folder_id: FolderID | set[FolderID],
     product_name: ProductName,
+    # updatable columns
+    name: str | UnSet = _unset,
+    parent_folder_id: FolderID | None | UnSet = _unset,
+    trashed_at: datetime | None | UnSet = _unset,
+    trashed_explicitly: bool | UnSet = _unset,
 ) -> FolderDB:
+    """
+    Batch/single patch of folder/s
+    """
+    # NOTE: exclude unset can also be done using a pydantic model and dict(exclude_unset=True)
+    updated = as_dict_exclude_unset(
+        name=name,
+        parent_folder_id=parent_folder_id,
+        trashed_at=trashed_at,
+        trashed_explicitly=trashed_explicitly,
+    )
+
+    query = (
+        (folders_v2.update().values(modified=func.now(), **updated))
+        .where(folders_v2.c.product_name == product_name)
+        .returning(*_SELECTION_ARGS)
+    )
+
+    if isinstance(folder_id, set):
+        if len(folder_id) == 0 or len(folder_id) > _MAX_ITEMS_UPDATE:
+            msg = f"Number of items for batch update is out-of-range, got {len(folder_id)=}"
+            raise ValueError(msg)
+
```

Comment:
why this limit?

Reply:
The limit is a hard precaution for now. 
I want to start exposing batch counterparts for some of the operations we have, e.g. `delete` and `batchDelete`, `trash` and `batchTrash`. I have been exploring different options but all of them (like in the pagination problem) need to define a limit on the batch items.  I already drafted a pagination mechanism for batch operations but I would like to add it in a separate PR

Reply:
then I would have a test, otherwise it's a bit like something that will come out of the blue

Reply:
https://docs.sqlalchemy.org/en/20/orm/large_collections.html#bulk-update-and-delete-of-items

ok

Reply:
I had a look at the link and this applies the ORM part of `sa`. I wonder if there is an equivalent for `sa.core`. So far, i did not find it. I will continue looking and will introduce this in a second iteration.